multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/cf/CloudControllerClientProvider.java

```diff
@@ -35,44 +38,28 @@
      * @param userName the user name associated with the client
      * @param org the organization associated with the client
      * @param space the space associated with the client
-     * @param processId the processId for the client
+     * @param correlationId of the process which is used to tag HTTP requests
      * @return a CF client for the specified access token, organization, and space
      */
-    public CloudControllerClient getControllerClient(String userName, String org, String space, String processId) {
+    public CloudControllerClient getControllerClient(String userName, String org, String space, String correlationId) {
         try {
-            return getClientFromCache(userName, org, space, processId);
+            return getClientFromCache(userName, org, space, correlationId);
         } catch (CloudOperationException e) {
             throw new SLException(e, Messages.CANT_CREATE_CLIENT_2, org, space);
         }
     }
 
-    /**
```

Comment:
I do not see advantage of the removal. In this way it is expected you force the java client code to pass null values, which should be handled afterwards.
Check https://stackoverflow.com/questions/4535112/is-null-arguments-a-bad-practice

Reply:
I will think of better way to do what is intended.